storage/src/adb/current.rs

```diff
@@ -364,10 +329,7 @@ impl<
             .await?;
         self.status.sync(&mut grafter).await?;
 
-        let target_prune_loc = self
-            .any
-            .inactivity_floor_loc
-            .saturating_sub(self.any.pruning_delay);
+        let target_prune_loc = self.any.inactivity_floor_loc;
```

Comment:
Do we need to create a variable here still?

Reply:
nope fixed